g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/CommonUtility.java

```diff
@@ -5,35 +5,39 @@
 package com.vmware.vip.i18n.api.base.utils;
 
 import com.vmware.i18n.utils.CommonUtil;
+import com.vmware.vip.common.constants.ConstantsChar;
 import com.vmware.vip.common.utils.CategoriesEnum;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class CommonUtility {
   private CommonUtility() {}
+
     /**
-     * Check param
-     * @param categories
+     * Get categories by CategoriesEnum
+     * @param scope
+     * @param perfectMatchFlag
+     * If perfectMatchFlag = true and not found from the CategoriesEnum, return null
+     * If perfectMatchFlag = false and not found from the CategoriesEnum，return the original value
+     *
      * @return
      */
-    public static boolean checkParams(List<String> categories, String... args){
-        List<String> newCategories = new ArrayList<>();
+    public static List<String> getCategoriesByEnum(String scope, boolean perfectMatchFlag){
```

Comment:
I think name the parameter as 'perfectMatch' is enough.

Reply:
fixed